eth/ethconfig/gen_config.go

```diff
@@ -21,12 +21,16 @@ func (c Config) MarshalTOML() (interface{}, error) {
 		Genesis                              *core.Genesis `toml:",omitempty"`
 		NetworkId                            uint64
 		SyncMode                             downloader.SyncMode
+		HistoryMode                          HistoryMode
```

Comment:
Just in case, this file was generated by `go run github.com/fjl/gencodec -type Config -formats toml -out gen_config.go` right?

Reply:
Yes, correct. (`ctrl+r` confirmed)